beast/plotting/plot_toothpick_details.py

```diff
@@ -123,3 +122,58 @@ def plot_toothpick_details(asts_filename, seds_filename, savefig=False):
         fig.savefig("{}.{}".format(basename, savefig))
     else:
         plt.show()
+
+def fetch_filters(filename):
+    """
+    For any FITS table file (gst, ast, etc.), collect the filters based on the
+    columns containing "VEGA".
+
+    Parameters
+    ----------
+    filename : str
+        filename containing VEGA measurements
+
+    Returns
+    -------
+    filters : list
+        list of full filter names
+
+    """
+
+    # all possible filters
+    gst_filter_names = [
+        "F225W",
+        "F275W",
+        "F336W",
+        "F475W",
+        "F814W",
+        "F110W",
+        "F160W",
+        "F657N",
+    ]
+    beast_filter_names = [
+        "HST_WFC3_F225W",
+        "HST_WFC3_F275W",
+        "HST_WFC3_F336W",
+        "HST_WFC3_F475W",
+        "HST_WFC3_F814W",
+        "HST_WFC3_F110W",
+        "HST_WFC3_F160W",
+        "HST_WFC3_F657N",
+    ]
```

Comment:
This is not a complete list.  For example, the PHAT/SMIDGE observations would fail here as the optical was taken with ACS not WFC3.   Is there anyway to test if there are other Vega filters in the AST file that are not included in these lists and raise a warning/error?